### DISCUSSION TITLE
Render spriteheet: Padding setting ignored when choosing individual frames.

### DISCUSSION THREAD

**Discussion by ErwinBr** [opentoonz/opentoonz#6471] [Ideas]
### Padding setting ignored when choosing individual frames.

When rendering a spritesheet, I found that the padding setting is not respected if you choose individual frames as the output format. It works fine in the other options (e.g. Grid).

Is there a reason for that, or is it an oversight?

The reason I'm asking is because I'm having "bleeding" issues in my game engine that could've been solved with padding. And the sprites are too big this particular project to export all of them into a single sheet.

![Screenshot 2024-04-01 at 15 19 09](https://github.com/opentoonz/opentoonz/assets/46350573/6cf3f2ac-6b6b-40e4-b543-b938617e71b6)

**Comment by RodneyBaker** [comment]
Have you tried this with the 'Trim Empy Space' option toggle off?

**Comment by ErwinBr** [comment]
Hi @RodneyBaker, thanks for looking into it. I did not check it without that option off, but the thing is, I do actually need to trim the empty spaces. So far it's been working great in SpriteSheet method, which I use most of the time. I wish it would work the same and be consistent with Individual frames. I now have to resort to another app to fix this and it's slowing down the workflow. Would be fantastic if it can be addressed. Thanks again!

**Comment by RodneyBaker** [comment]
Converting this to a Discussion so others can hone in one what needs to be done.

**Reply by ErwinBr** [reply]
Thank you. Just wanted to mention that this is still relevant today and I'm looking forward to a solution.
